app/views/onboarding/index.html.erb

```diff
@@ -18,7 +18,7 @@
         <% end %>
 
         <div class="text-serif">
-          <%= c 'markdown', raw: Setting.onboarding_page %>
+          <%= c 'markdown', raw: Setting.onboarding_page, class: 'rich-content' %>
```

Comment:
@tillprochaska you probably know a better way to do this, but when I updated the `markdown` component to use the `attrs` instead of `class_attrs`, I could not set a default class as we had been doing. Since it is used only in 1, now 2, places, I just added the class in the call to render the component.

Reply:
Added a comment/suggestion above :)